### PR TITLE
Feature/update user panel component

### PR DESCRIPTION
**Pull request by DimaPolishuk1995**

### CODE REVIEW

File: src/components/containers/NavSearch/NavSearch.js [hunk 1]
```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import Dropdown from 'simple-react-dropdown'
+import SearchContent from '../SearchContent/SearchContent'
```
Comment (polesskiy-dev) [inline src/components/containers/NavSearch/NavSearch.js]: Should be empty line between. Did you add linter?

File: src/components/containers/NavSearch/NavSearch.js [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+  render () {
+    return(
+      <div className="wrap-search wrap-header-search">
+        <div className="header-search">
+          <div className="control-group left control-search-select dropdown">
+            <Dropdown content={<SearchContent />}>
+              <button className="btn btn-dropdown-toggle btn-search-toggle"><i className="btn-icon fa fa-bars"/></button>
+            </Dropdown>
+          </div>
+          <button className="btn btn-search"><i className="btn-icon fa fa-search"/></button>
```
Comment (polesskiy-dev) [inline src/components/containers/NavSearch/NavSearch.js]: we already have special component for it

File: src/components/containers/NavSearch/NavSearch.js [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
+          <div className="wrap-search-holder">
+            <div className="search-holder">
```
Comment (polesskiy-dev) [inline src/components/containers/NavSearch/NavSearch.js]: We should create separate component for it. Can we use <Input/> from react-bootstrap?

File: src/components/containers/NotificationContent/NotificationContent.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const NotificationContent = () => {
```
Comment (polesskiy-dev) [inline src/components/containers/NotificationContent/NotificationContent.js]: It should be somewhere in representation/temprorary/

File: src/components/containers/UserAccountPanel/UserAccountPanel.js [hunk 5]
```diff
@@ -1,31 +1,48 @@
 import React from 'react';
 import PTButton from '../PTButton/PTButton';
+import UserPanelItem from '../UserPanelItem/UserPanelItem';
```
Comment (polesskiy-dev) [inline src/components/containers/UserAccountPanel/UserAccountPanel.js]: mixed absolute and relative imports. Use linter, please.

File: src/components/containers/UserAccountPanel/UserAccountPanel.js [hunk 6]
```diff
@@ -1,31 +1,48 @@
...
+             <PTButton className="btn-header"
+               children={(
+                 <i className="fa fa-search"/>
+               )}
+             />
+           )}
+        />
+        <UserPanelItem className="user-panel-item dropdown"
+           children={(
+             <Dropdown content={<NotificationContent />}>
```
Comment (polesskiy-dev) [inline src/components/containers/UserAccountPanel/UserAccountPanel.js]: And how we're going to handle dropdown item selection?

Comment (DimaPolishuk1995) [inline src/components/containers/UserAccountPanel/UserAccountPanel.js]: I look at a live site, there was no processing of the notification field

Comment (polesskiy-dev) [inline src/components/containers/UserAccountPanel/UserAccountPanel.js]: ok

**Pull request merged**